middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -915,14 +972,15 @@ let after_recursive_let_cont_body_rebuilt cont handlers ~uenv_without_cont
     ~cost_metrics_of_handlers ~after_rebuild body uacc =
   (* We are passing back over a binder, so remove the bound continuation from
      the free name information. *)
+  let free_names_of_body = UA.name_occurrences uacc in
```

Comment:
It's not clear to me why this `uacc` does not contain the free names of the body **and** handler ?